beastify/popup/choose_beast.css

```diff
@@ -2,15 +2,31 @@ html, body {
   width: 100px;
 }
 
-.beast {
+.button {
   margin: 3% auto;
   padding: 4px;
   text-align: center;
   font-size: 1.5em;
-  background-color: #E5F2F2;
   cursor: pointer;
 }
 
-.beast:hover {
+.button:hover {
   background-color: #CFF2F2;
 }
+
+.beast {
+  background-color: #E5F2F2;
+}
+
+.clear {
+  background-color: #E74C3C;
```

Comment:
It's nice for the button to be a different color from the others, but this is a bit bright. #FBFBC9 is a bit more subtle.